api_proto.go

```diff
@@ -345,7 +345,7 @@ func (p *Progress) ToProto() *proto.Progress {
 	}
 }
 
-func SearchResultFromProto(p *proto.SearchResponse) *SearchResult {
+func SearchResultFromProto(p *proto.SearchResponse, repoURls, lineFragments map[string]string) *SearchResult {
```

Comment:
I tweaked this conversion function since we removed repoURLs and lineFragments from the protobuf defnition - but we still have it around in the Zoekt struct type.